src/app/hooks/useNetworkFees.ts

```diff
@@ -25,39 +34,62 @@ const formatFee = (fee: string, rate: BigNumber) => {
     fiat: CurrencyFormatter.cryptoToCurrency(cryptoAmount, rate, {
       decimals: 2,
     }),
-    crypto: cryptoAmount.decimalPlaces(4).toFixed(4),
+    crypto: Number.parseFloat(
```

Comment:
May I ask the reasoning behind this? This could reformat value in scientific notation, fyi